extensions/wikia/AdEngine/js/lookup/lookupFactory.js

```diff
@@ -100,6 +100,7 @@ define('ext.wikia.adEngine.lookup.lookupFactory', [
 			getSlotParams: getSlotParams,
 			hasResponse: hasResponse,
 			trackState: trackState,
+			onResponse: onResponse,
```

Comment:
I wouldn't like to expose this method especially if it's only for tests. It should be strictly private. If I'm not wrong there is another approach to call it within tests. It's available in `window.OX_dfp_options.callback`. What do you think?